allure-web/src/components/execution/ExecutionView.js

```diff
@@ -29,10 +29,26 @@ class ExecutionView extends View {
         attachmentEl.parents('.step').addClass('step_expanded');
     }
 
+    calculateSteps(steps, key) {
+        return steps.map(step => {
+            var steps = this.calculateSteps(step.steps, key);
+            step[key + 'Count'] = step[key].length + steps.reduce((count, cur) => { return count + cur[key + 'Count'];}, 0);
+            step.steps = steps;
```

Comment:
is this a reference to the parent?
Would it be better to call it `parent` then?